contracts/PlotValuation.sol

```diff
@@ -274,7 +274,11 @@ contract PlotValuation is AbstractApplication {
 
     a.firstValuation = _valuation;
 
-    changeApplicationStatus(a, ApplicationStatus.VALUATED);
+    if (a.firstValuation == a.secondValuation) {
+      changeApplicationStatus(a, ApplicationStatus.CONFIRMED);
+    } else {
+      changeApplicationStatus(a, ApplicationStatus.VALUATED);
```

Comment:
Not sure that this check required

Reply:
Nikolay asked for this

Reply:
Ok, I got this. This is for the following case:
* 1st appraiser valuation is 2 (=> `VALUATED`)
* 2nd appraiser valuation is 3 (=> `REVERTED`)
* 1st appraiser valuation is 3 (=> `CONFIRMED`)